Sync/CleartextPayloadJSON.swift

```diff
@@ -4,19 +4,21 @@
 
 import Foundation
 import Shared
+import SwiftyJSON
 
-open class BasePayloadJSON: JSON {
+open class BasePayloadJSON {
+    var _json: JSON
```

Comment:
Don't think we need to prefix the variable name with an underscore here? If we want to make it a private setter we can use `private(set) var json: JSON`